### DISCUSSION TITLE
Vscode and Eslint linting typescript rules on js files on non-ts prpject???

### DISCUSSION THREAD

**Discussion by renato-ll** [eslint/eslint#15438] [Q&A]
Help folks,

I even dont have ts installed on my project!
Why eslint keeps showing ts related warnings/errors at my js files?
Is there any way to completly disable typescript eslint linting at my JAVASCRIPT NODE project?

Im trying for more than 12 hours!
(Ive tryied every redit, stackoverflw, posts a d github issues)

Please help me, i just want a simple js script linting at vscode!

To be more complicated:
A simple vscode lint for js nodejs setup, pure esm setup(type=module), and eslint aware about nodejs running as type=module without commonsjs, and ts related warning disabled

Its basicly impossible to do that?

**Comment by aladdin-add** [comment]
hi, eslint does not support linting ts (without third party plugins). Have you configured something like typescript-eslint?

**Reply by renato-ll** [reply]
Well, i was using     // "checkJs": true on jsconfig.json

i believe it was it
